.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: php
+php:
+  - '7.3'
+dist: bionic
+env:
+  - TGWF_CONFIG_FILE_PATH=/config.ci.mysql.yml
```

Comment:
we run one for mysql and one for postgres. However both do run the phpstan stuff. That might be a bit wastefull, something to optimise later?

Reply:
One to optimise later, I think